server/src/main/java/fr/uga/l3miage/tp2/exo3/models/CommandeEntity.java

```diff
@@ -0,0 +1,18 @@
+package fr.uga.l3miage.tp2.exo3.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.util.Set;
+
+@Entity
+public class CommandeEntity {
+    @Id
+    private Long id ;
+    private double montantTotal ;
+    @ManyToOne
+    private ClientEntity client ;
+    @OneToMany
+    private Set<ProduitEntity> produits ;
```

Comment:
il faut un `@JoinColumn` sinon vous créer une table d'association

```suggestion
    @OneToMany
    @JoinColumn(name = "id_produit",referencedColumnName = "id")
    private Set<ProduitEntity> produits ;
```